bqn-comint.el

```diff
@@ -101,28 +79,42 @@ If it doesn't exist, create and return it; else, return the existing one."
       (switch-to-buffer-other-window *bqn-process-buffer-name*)
       (bqn-inferior-mode)
       (set-input-method "BQN-Z"))))
-
-(defun bqn-process-execute-region (start end &optional dont-follow)
-  "Send the current region to the bqn-process-session.
-
-When DONT-FOLLOW is non-nil, maintain focus on the buffer where the function was called from."
+(defun bqn-process-execute-region (start end &optional dont-follow return-output)
```

Comment:
1. Add a blank line between functions;
2. Docstring them:
   
   https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html

3. More generally, [follow the guide](https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html).